docs/BR.md

```diff
@@ -1901,14 +2025,15 @@ For every valid Certification Path (as defined by RFC 5280, Section 6):
 * For each CA Certificate in the Certification Path, the encoded content of the Subject Distinguished Name field of a Certificate SHALL be byte-for-byte identical among all Certificates whose Subject Distinguished Names can be compared as equal according to RFC 5280, Section 7.1, and including expired and revoked Certificates.
 
 #### 7.1.4.2 Subject Information - Subscriber Certificates
+
 By issuing the Certificate, the CA represents that it followed the procedure set forth in its Certificate Policy and/or Certification Practice Statement to verify that, as of the Certificate's issuance date, all of the Subject Information was accurate. CAs SHALL NOT include a Domain Name or IP Address in a Subject attribute except as specified in [Section 3.2.2.4](#validation-of-domain-authorization-or-control) or [Section 3.2.2.5](#authentication-for-an-ip-address).
 
 Subject attributes MUST NOT contain only metadata such as '.', '-', and ' ' (i.e. space) characters, and/or any other indication that the value is absent, incomplete, or not applicable.
 
 ##### 7.1.4.2.1 Subject Alternative Name Extension
 
-__Certificate Field:__ `extensions:subjectAltName`  
-__Required/Optional:__ Required  
```

Comment:
These are the structurally significant ones that force line breaks.

Reply:
Oh, and it is CommonMark compliant, I guess - https://spec.commonmark.org/0.17/#hard-line-breaks

Reply:
:+1: Think I've restored all of these.